src/nnbench/types.py

```diff
@@ -149,11 +182,18 @@ class FilePathArtifactLoader(ArtifactLoader):
     Parameters
     ----------
     path : str | os.PathLike[str]
-        The path to the artifact, which can include a protocol specifier (like 's3://') for remote access.
```

Comment:
How many columns does a docstring line have at maximum?
Or do you just linebreak after every sentence?